modules/qualitycontrol/QualityController.ts

```diff
@@ -380,7 +377,7 @@ export class QualityController {
                     trackStats: [ ssrcStats ]
                 };
 
-                statsPerTrack.set(track.rtcId, existingStats);
+                statsPerTrack.set(trackId, existingStats);
```

Comment:
Why are we setting the string as a key, which then forces us to use `.toString()` everywhere?

Reply:
I initially declared the key for the encode time stats to be a string and later realized that `rtcId` is a number. I changed it back. At that time I thought a string made more sense if its a number/string but I see that its a number always. I changed it to a number number.